test/pytest/mocktransport.py

```diff
@@ -120,9 +122,13 @@ async def assert_data(self, expected_channel: str, expected_data: bytes) -> None
         assert channel == expected_channel
         assert data == expected_data
 
-    async def assert_msg(self, expected_channel: str, **kwargs: JsonValue) -> JsonObject:
+    async def assert_msg(self, expected_channel: str, absent_keys: Optional[List] = None,
+                         **kwargs: JsonValue) -> JsonObject:
         msg = await self.next_msg(expected_channel)
         assert msg == dict(msg, **{k.replace('_', '-'): v for k, v in kwargs.items()}), msg
+        if absent_keys:
```

Comment:
You could drop this check if you remove the possibility of this being `None` and use `()` instead.